Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -1235,6 +1266,17 @@ extension FirefoxHomeViewController {
                                      object: .firefoxHomepage,
                                      value: .customizeHomepageButton)
     }
+
+    @objc func changeHomepageWallpaper() {
+        // TODO: Roux - This function will with the action when wallpaper feature is
```

Comment:
Since I had another suggestion, might as well suggests this here haha
```suggestion
        // TODO: Roux - This function will be implemented with the action when wallpaper feature is
```